app/code/core/Mage/Adminhtml/Block/Widget/Grid/Column/Filter/Date.php

```diff
@@ -50,15 +50,15 @@ public function getHtml()
         $htmlId = $this->_getHtmlId() . microtime(true);
         $format = $this->getLocale()->getDateStrFormat(Mage_Core_Model_Locale::FORMAT_TYPE_SHORT);
         $html = '<div class="range"><div class="range-line date">'
-            . '<span class="label">' . Mage::helper('adminhtml')->__('From').':</span>'
```

Comment:
there should be spaces around dot, here and in other plces

Reply:
done in PR 1026.